### ISSUE TITLE
active_record_options - establish_connection creates connection per model

### ISSUE THREAD

**Issue by pk1m** [doorkeeper-gem/doorkeeper#1322]
### Steps to reproduce
I'm currently connecting to doorkeeper models on a second database so I can directly verify tokens.  I'm using `active_record_options` to call `establish_connection` to the other database but it creates a new connection per model.  Is it possible to share a connection?

### Expected behavior
Create one connection.

### Actual behavior
Creates 3 connections, one per doorkeeper model

### System configuration
You can help us to understand your problem if you will share some very
useful information about your project environment (don't forget to
remove any confidential data if it exists).

**Doorkeeper initializer**:

```ruby
# config/initializers/doorkeeper.rb
Doorkeeper.configure do
  active_record_options(
    establish_connection: Rails.configuration.database_configuration["#{Rails.env}_realsavvy"],
  )
end
```

**Ruby version**: `2.6.3`

**Gemfile.lock**: ` doorkeeper (4.2.6)`

<details>
  <summary>Gemfile.lock content</summary>
  
```
Place your Gemfile.lock content here
```
</details>

**Comment by nbulaj** [comment]
I think this issue could be easily resolved by introducing changes from #1327

**Comment by nbulaj** [comment]
Hi @pk1m . You can now use cusomized Doorkeeper models starting from Doorkeeper 5.3 so you can setup any connection you need. More info here https://doorkeeper.gitbook.io/guides/configuration/models

Could we now close the issue?
